packages/transaction-pay-controller/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/transaction-pay-controller",
-  "version": "10.5.0",
+  "version": "10.6.0",
```

Comment:
Could you explain why you are bumping this controller as well?

Reply:
from @bfullam 

> That bump happened as part of the release creation script, not intentional on my part.

Let me check if I can revert those changes

Reply:
There must be some confusion then. The release creation script does not automatically bump anything.